### DISCUSSION TITLE
[Frontend] Allo-PyTorch Enhancement

### DISCUSSION THREAD

**Discussion by chhzh123** [cornell-zhang/allo#212] [Projects]
Currently, Allo has already had a compilation flow from PyTorch, but the generated program is not yet high-performance. We need to enhance automatic schedule composition and incorporate linalg/tensor dialects for better representing the neural network operations.

**Comment by LarryFan** [comment]
10/21 Update
- Busy past 2 weeks with projects, exams, etc.
- Installed, Torch, Transformers to run example models
- Ran bert, toy, self-attention, mlp
- Recorded + Analyzing graphs and ASTs

**Comment by LarryFan** [comment]
11/4

Debugged MLIR and Vitis HLS to run benchmarks
Working on loop unroll and pipelining integration + test

**Comment by chhzh123** [comment]
@LarryFan Please check #248 that I've exposed the HLS target to the frontend
